<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>person</class>
 <widget class="QWidget" name="person">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/bird.png</normaloff>:/images/bird.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>300</width>
        <height>638</height>
       </rect>
      </property>
      <attribute name="label">
       <string>我的好友</string>
      </attribute>
      <layout class="QFormLayout" name="vlayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="hymEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="addf_btn">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>添加好友</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
